### DISCUSSION TITLE
Website down?

### DISCUSSION THREAD

**Discussion by knopp10000** [Tichau/FileConverter#207] [General]
I can't reach "https://file-converter.org/".

**Comment by andrasfuchs** [comment]
It's down for me too. The last snapshot is from last November: https://web.archive.org/web/20211103175752/https://file-converter.org/

**Comment by KyGuy2002** [comment]
Don't mean to necro, but the site at https://www.file-converter.org/ doesn't work, but https://file-converter.org/ does.  Looks like s simple DNS issue, but all links on GitHub use the www, so they don't work.

**Comment by Tichau** [comment]
The website is now at the following adress: https://file-converter.io
